src/Core/Models/Business/OrganizationLicense.cs

```diff
@@ -133,10 +135,10 @@ public OrganizationLicense()
     /// <summary>
     /// Represents the current version of the license format. Should be updated whenever new fields are added.
     /// </summary>
-    private const int CURRENT_LICENSE_FILE_VERSION = 10;
+    private const int CURRENT_LICENSE_FILE_VERSION = 12;
```

Comment:
This constant is intentionally set to be a version behind. https://bitwarden.slack.com/archives/C02LPDKPML6/p1668134749839749

Reply:
Ah, I should have realized it was behind by one for some reason, thanks for catching that!

Reply:
Set `CURRENT_LICENSE_FILE_VERSION=10` and added comment explaining why its one behind the actual current version to hopefully catch this for anyone else in the future.